x-pack/plugins/security_solution/common/api/detection_engine/prebuilt_rules/get_prebuilt_rules_and_timelines_status/get_prebuilt_rules_and_timelines_status_route.schema.yaml

```
@@ -5,6 +5,9 @@ info:
 paths:
   /api/detection_engine/rules/prepackaged/_status:
     get:
+      x-labels:
+        - ess
+        - serverless
```

Nit: this format would be less verbose

```suggestion
      x-labels: ['ess', 'serverless']
```

---

YAML allows even simple form without quotes

```yaml
 x-labels: [ess, serverless]
```